CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `libherokubuildpack`:
   - `MappedWrite::unwrap` for getting the wrapped `Write` back out. ([#765](https://github.com/heroku/libcnb.rs/pull/765))
+  - Added build `buildpack_output` module. This will help buildpack authors provide consistent and delightful output to their buildpack users ([#721](https://github.com/heroku/libcnb.rs/pull/721))
```

Comment:
This will need rebasing on `main` and the changelog entry adding to the new `unreleased` section, now that 0.18.0 has been released.

Reply:
Conflicts in cargo.toml for features that were added then removed, all while I'm trying to not screw the PR history here so we can turn around and squash merge it in.  

I'll wait until all comments are final before trying.